src/RoadRegistry.Editor.ProjectionHost/Program.cs

```diff
@@ -179,12 +179,15 @@ public static async Task Main(string[] args)
                                     new DbContextOptionsBuilder<EditorContext>()
                                         .UseSqlServer(
                                             hostContext.Configuration.GetConnectionString(WellknownConnectionNames.EditorProjections),
-                                            options => options.EnableRetryOnFailure()
+                                            options => options
+                                                .EnableRetryOnFailure()
+                                                .UseNetTopologySuite()
                                         ).Options)
                         )
                         .AddSingleton(sp => new ConnectedProjection<EditorContext>[]
                         {
                             new OrganizationRecordProjection(sp.GetRequiredService<RecyclableMemoryStreamManager>(), WindowsAnsiEncoding),
+                            new MunicipalityGeometryRecordProjection(),
```

Comment:
Doubt we need to use the word record here - `record` is used to denote something that will appear as a record in a dbase / shape file, which is not the case here.

Reply:
Ah, thanks for the clarification, I didn't get when to use record or not.